Source/WTF/wtf/text/StringImpl.h

```diff
@@ -1047,7 +1047,7 @@ template<typename CharacterType> ALWAYS_INLINE RefPtr<StringImpl> StringImpl::tr
         output = { };
         return nullptr;
     }
-    StringImpl* result;
+    SUPPRESS_UNCOUNTED_LOCAL StringImpl* result;
 
     result = (StringImpl*)StringImplMalloc::tryMalloc(allocationSize<CharacterType>(length));
```

Comment:
Seems we should initialize the variable right away, above.

Reply:
Sure, although that'll make [[clang::suppress]] apply for the whole line including the call to tryMalloc...